packages/twenty-front/folderStructure.json

```
@@ -63,12 +51,12 @@
         { "name": "internal", "ruleId": "hooksLeafFolderRule" }
       ]
     },
+
     "doNotCheckLeafFolderRule": {
-      "folderRecursionLimit": 1,
       "children": [{ "name": "*" }, { "name": "*", "children": [] }]
```

It can be shortened to:
```suggestion
      "children": []
```